validator/db/kv/manage_test.go

```diff
@@ -171,15 +171,15 @@ func prepareStore(store *Store, pubKeys [][48]byte) (*storeHistory, error) {
 	return &history, nil
 }
 
-func prepareStoreProposals(store *Store, pubKeys [][48]byte) (map[[48]byte]bitfield.Bitlist, error) {
-	proposals := make(map[[48]byte]bitfield.Bitlist)
+func prepareStoreProposals(store *Store, pubKeys [][48]byte) (map[[48]byte][]byte, error) {
+	proposals := make(map[[48]byte][]byte)
 
 	for i, key := range pubKeys {
-		proposalHistory := bitfield.Bitlist{byte(i), 0x00, 0x00, 0x00, 0x01}
-		if err := store.SaveProposalHistoryForEpoch(context.Background(), key[:], 0, proposalHistory); err != nil {
+		signingRoot := bytesutil.PadTo([]byte{byte(i)}, 32)
```

Comment:
this is confusing, why is the signing root the index of the pubkey ?

Reply:
Just to have different values